persist-cli-tests/src/test/resources/test-src/input/tool_test_generate_5/generated/entities/persist_client.bal

```diff
@@ -19,7 +19,7 @@ public client class EntitiesClient {
     private final map<persist:SQLClient> persistClients;
 
     private final record {|persist:Metadata...;|} metadata = {
-        "medicalneed": {
+        "MEDICAL_NEED": {
```

Comment:
why we need to change this

Reply:
i this example we run the persist generate command on already generated project. To differentiate between the already existing client and the newly generated client this line was changed